source/getting-started/ready-for-symbol-launch.rst

```diff
@@ -0,0 +1,177 @@
+
+This guide explains:
+
+- How to enroll in the different :doc:`Reward Programs <../concepts/reward-programs>`, created to **incentivize nodes** to come online as soon as possible.
+
+- How to configure your node so that it **starts harvesting as quickly as possible** when |codename|'s network launches.
+
+Snapshot and launch
+===================
+
+As you should be well aware by now, |codename|'s launch will happen in two phases.
+
+1. **Snapshot**: When the NIS1 chain reaches block height 3'105'500 a **snapshot** of the contents of all NIS1 accounts will be taken. `This is expected to happen on 12 March 2021, at around 00:50 UTC <https://nem.io/#symbol-countdown>`__.
+
+2. **Launch**: Afterwards, on March 15th (exact time to be determined), the new |codename| network will **launch** meaning that the nodes managed by NEM Group will be brought online and they will start adding blocks to the shinny new |codename| blockchain.
+
+These two concepts are used throughout this guide.
+
+*********
+Enrolling
+*********
+
+If you want to participate in either the :ref:`early-adoption-node-program` or the :ref:`ecosystem-node-program` you need to enroll in them **before the snapshot**. This section explains how to do this.
+
+If you are interested only in the :ref:`supernode-program` or the :ref:`voting-node-program` (or if you are not interested in any of the reward programs) you don't need to enroll beforehand so you can skip this section and jump to :ref:`getting-ready-launching-your-node`.
+
+.. _getting-ready-prerequisites:
+
+Prerequisites
+=============
+
+- You must have a `NIS1 <https://nemplatform.com/>`__ account. `NIS1 <https://nemplatform.com/>`__ is NEM's current blockchain, which is used in this process to **register all requests** before |codename|'s launch. The easiest way to create an account is through the `NEM Desktop Client <https://nemplatform.com/wallets/#desktop>`__ (also known as the **NEM Nano Wallet**) and click the **SIGN UP** button at the top-right.
+
+- Your NIS1 account must be `opted-in <https://nemplatform.com/symbol-migration/>`__. The opt-in process creates a new |codename| account that will receive a copy of the NIS1 account's **tokens** once |codename| launches. Again, the easiest way to do this is through the **Nano Wallet's opt-in module**:
+
+  - Make sure you obtain a **Paper Wallet** for the new |codename| account, or, at least, its **private key**.
+  - Do not worry about opting-in the **VRF key**, it is currently being ignored (there is no problem if you already included it).
+
+- Your NIS1 account must hold at least **10'000 XEMs** at the snapshot block height (3'105'500). :ref:`Only accounts holding this amount of XYMs can harvest <account_eligibility>` on the |codename| network so only accounts holding this amount at the snapshot can benefit from the Reward Programs.
```

Comment:
10000 refers to vested or total balance?
Q: the vested balance is a number that is preserved in the transition XEM->SYM?

Reply:
10'000 XEMs will turn into 10'000 XYMs, and that's what's needed to harvest on Symbol.
Whether the 10'000 XEMs are vested or not does not matter.
I would not introduce the concept of "vesting" in this guide since these are the Symbol docs, and vesting does not appear anywhere else.